src/main/java/dev/stratospheric/cdk/Network.java

```diff
@@ -302,10 +302,25 @@ private void createLoadBalancer(
         .open(true)
         .build());
 
-
-      httpsListener.addTargetGroups("https-dummy", AddApplicationTargetGroupsProps.builder()
+      httpsListener.addTargetGroups("https-defaultTargetGroup", AddApplicationTargetGroupsProps.builder()
         .targetGroups(Collections.singletonList(dummyTargetGroup))
         .build());
+
+      ListenerAction redirectAction = ListenerAction.redirect(
+        RedirectOptions.builder()
+          .port("443")
+          .build()
+      );
+      ApplicationListenerRule applicationListenerRule = new ApplicationListenerRule(
+        this,
+        "HttpListenerRule",
+        ApplicationListenerRuleProps.builder()
+          .listener(httpsListener)
```

Comment:
Shouldn't the rule be attached to the `httpListener` instead? - so it redirects to the `httpsListener`

Reply:
Yes, you're right. Thanks a lot for pointing that out. I'll update the code accordingly.